client/my-sites/exporter/section-export.jsx

```diff
@@ -49,7 +49,7 @@ const SectionExport = ( { isJetpack, canUserExport, site, translate } ) => {
 			<Fragment>
 				<FormattedHeader
 					className="exporter__section-header"
-					headerText={ translate( 'Export Your Content' ) }
+					headerText={ translate( 'Export Content' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **34** times:
`translate( 'Export content' )` *ES Score: 13*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/card-headings-form-labels-sentence-case)

Reply:
Should this be a lower case C?

```suggestion
					headerText={ translate( 'Export content' ) }
```

Reply:
No. Page headings will remain title case.